motes.md

```diff
@@ -0,0 +1,42 @@
+* `BLS_WITHDRAWAL_PREFIX`
```

Comment:
Loving your motes :) Will comment on the various points later today 😂

Reply:
Oh no! I accidentally pushed. Haaa